### ISSUE TITLE
Keys tickets and fare cards missing 5.1.3.16 

### ISSUE THREAD

**Issue by benjamin-rolleman** [aaact-aatia/a11y-ict-wizard#27]
Keys tickets and fare cards needs to also select 5.1.3.16 

This bug is currently planning to be fixed with the inclusion of the WCAG French into the wizard project which should conclude by end of January 2023.
